product_docs/docs/pgd/5/pgdlifecycle.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: PGD lifecycle
+---
+
+## Introduction
+
+As part of any planning process, it is essential to know the supported platforms and planned supported life of the application. This section provides those details for PGD 5.x.
+
+## PGD 5.x Lifecycle
+
+| Event | When |
+| --- | --- |
+| General availability | 21 Feb 2023 |
+| Standard support ends | 18 months after the release of 6.0 |
+
+
+## PGD 5.x Supported CPU architectures and OS
+
+### Linux x86-64 (Amd64)
+
+| Operating System | Versions&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  |
+| ---------------- | -------- |
+| RHEL | 9/8/7 |
+| OL | 9/8/7 |
+| RockyLinux/AlmaLinux | 9/8 |
+| CentOS | 7 |
+| SLES | 15SP4/12SP5 |
```

Comment:
Let's use official product names, not acronyms: "SUSE Linux Enterprise Server (SLES)"

Reply:
So also expand out RHEL and Oracle Linux

Reply:
And you might want to tell whoever does the Product Compat page on the site because it's all like that:

<img width="385" alt="Screenshot 2023-10-06 at 11 32 05" src="https://github.com/EnterpriseDB/docs/assets/126472455/9f1c91e6-cded-4434-8c30-753415a28e8a">